### PR TITLE
PCHR-2637: Replace assignment with workflow

### PR DESCRIPTION
**Pull request by AkA84**
This PR simply swaps any occurrence of the word "Assignment" with the word "Workflow" throughout the extension

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/templates/CRM/Tasksassignments/Reminder/DailyReminderActivity.tpl [hunk 1]
```diff
@@ -16,7 +16,7 @@
                     </td>
```
Comment (igorpavlov) [inline uk.co.compucorp.civicrm.tasksassignments/templates/CRM/Tasksassignments/Reminder/DailyReminderActivity.tpl]: Could you please fix the indentation in this file? If you need to both automatic and manual fixes - please do not mix them and commit in separate commits.

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html [hunk 2]
```diff
@@ -234,20 +234,20 @@ <h4 class="modal-title" ng-switch="documentModal.isRole('admin')">
               </div>
             </div>
           </div>
-          <!-- Assignment -->
+          <!-- Workflow -->
```
Comment (igorpavlov) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html]: We do not store comments in the HTML code, do we?

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html [hunk 3]
```diff
@@ -109,7 +109,7 @@ <h4 class="modal-title">New Task</h4>
           </ui-select>
         </div>
       </div>
-      <!-- Status & Assignment-->
+      <!-- Status & Workflow-->
```
Comment (igorpavlov) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html]: Same as above

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html [hunk 4]
```diff
@@ -127,11 +127,11 @@ <h4 class="modal-title">New Task</h4>
             <span class="crm_custom-select__arrow"></span>
           </div>
         </div>
-        <!-- Assignment -->
+        <!-- Workflow -->
```
Comment (igorpavlov) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html]: Same as above

File: uk.co.compucorp.civicrm.tasksassignments/views/settings.html [hunk 5]
```diff
@@ -59,7 +59,7 @@ <h3 class="panel-title">Tasks Settings</h3>
                 <div class="form-group">
```
Comment (igorpavlov) [inline uk.co.compucorp.civicrm.tasksassignments/views/settings.html]: Could you please fix the indentation in this file?

**Comment by igorpavlov** [comment]
Please let me know how all places with "Workflow" word were found - manually or automatically (with grep or smth)?

**Comment by AkA84** [comment]
@igorpavlov via the IDE's search functionality

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html (hunk 2 above)
Comment (AkA84) [reply uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html]: no, but given how big and complex this file is (it has something like 10 levels of indentation), i thought that i could keep this legacy stuff until finally T&A is overhauled

Comment (igorpavlov) [reply uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html]: Ok

**Pull request merged**